service/model/release.go

```diff
@@ -26,14 +26,14 @@ type UpdateReleaseInput struct {
 }
 
 type Release struct {
-	ID           uuid.UUID
-	ProjectID    uuid.UUID
-	ReleaseTitle string
-	ReleaseNotes string
-	AuthorUserID uuid.UUID
-	CreatedAt    time.Time
-	UpdatedAt    time.Time
-	// TODO add source code link
+	ID            uuid.UUID
+	ProjectID     uuid.UUID
+	ReleaseTitle  string
+	ReleaseNotes  string
+	AuthorUserID  uuid.UUID
+	CreatedAt     time.Time
+	UpdatedAt     time.Time
+	GithubRelease GithubRelease
 }
 
 func NewRelease(input CreateReleaseInput, projectID, authorUserID uuid.UUID) (Release, error) {
```

Comment:
In case we stick to having `GithubRelease` as a separate type (I'm more inclined to rather flattening it out), is its creation going to be in a separate PR?..

Reply:
Yes, I would do it in separate PR.